### PR TITLE
Make perf scripts ready for insecure scale-out tests

### PR DESCRIPTION
**Pull request by chenqianfzh**
### scale-out verfication

Verified with a 100*3 denisty run.

All tenants succeeded.

```
 ----------------- Latency -----------------------
 Tenant arktos:
   saturation pods: perc50: 1.883689258s, perc90: 2.651879426s, perc99: 3.226859297s;
   latency pods: perc50: 1.922559492s, perc90: 2.713633502s, perc99: 3.165488625s;
 Tenant margaret:
   saturation pods: perc50: 1.868395916s, perc90: 2.611379031s, perc99: 3.186608656s;
   latency pods: perc50: 1.900346243s, perc90: 2.632520675s, perc99: 3.145233869s;
 Tenant zeta:
   saturation pods: perc50: 1.87341468s, perc90: 2.611740623s, perc99: 3.140060219s;
   latency pods: perc50: 1.887926712s, perc90: 2.643359116s, perc99: 3.098890526s;

 ----------------- Throughput -----------------------
Tenant arktos:  perc50: 20,  perc90: 20,  perc99: 22,  max: 22
Tenant margaret:    perc50: 20,  perc90: 20.2,  perc99: 22,  max: 22
Tenant zeta:    perc50: 19.8,  perc90: 20.2,  perc99: 20.2,  max: 20.2
```

Prometheus is working.

### scale-up verification

100-node density run succeeded.

```
 ----------------- Latency -----------------------
   saturation pods:  perc50: 1.800533709s, perc90: 2.53376106s, perc99: 2.973002785s; 
   latency pods: perc50: 1.768328641s, perc90: 2.565175408s, perc99: 2.914472244s; 
   
----------------- Throughput -----------------------
 perc50: 20,  perc90: 20,  perc99: 23.8,  max: 23.8
```

### CODE REVIEW

File: cluster/gce/config-default.sh [hunk 1]
```diff
@@ -540,11 +540,4 @@ PROXY_TAG="${SCALEOUT_PROXY_NAME:-scaleoutproxy}"
 # true if this is a resource partition cluster
 KUBERNETES_RESOURCE_PARTITION="${KUBERNETES_RESOURCE_PARTITION:-false}"
 # true if this is a tenant partition cluster
-KUBERNETES_TENANT_PARTITION="${KUBERNETES_TENANT_PARTITION:-false}"
-
-if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == true ]]; then
```
Comment (pdgetrf) [inline cluster/gce/config-default.sh:545]: what's the impact/reason of removing all these?

File: test/kubemark/start-kubemark.sh [hunk 2]
```diff
@@ -366,23 +346,4 @@ echo -e "Getting apiserver data partition status:" >&2
 echo
 echo -e "Getting ETCD data partition status:" >&2
 "${KUBECTL}" --kubeconfig="${TP_KUBECONFIG}" get etcd
-echo
-
-### for multiple tenant tests, set up the tenants on desired TP clusters
```
Comment (pdgetrf) [inline test/kubemark/start-kubemark.sh:371]: same question here. it's not obvious from the PR why these are removed .

**Comment by pdgetrf** [comment]
love seeing validation in the PR description. please also run the scale-up 100-node test to make sure of no regression.

File: cluster/gce/config-default.sh (hunk 1 above)
Comment (vinaykul) [reply cluster/gce/config-default.sh:545]: there should be no impact after merging with Sonya's change.
@chenqianfzh thanks for cleaning this up. Please remove it from config-test.sh as well if possible (no need to retest)

Comment (chenqianfzh) [reply cluster/gce/config-default.sh:545]: These code created two test tenants, arktos/zeta, when we were trying with 2 TPs. Yet the code was changed to create variable number of TPs, where two test tenants may not be enough. We need the tester to create the test tenants on the fly. Therefore this section of code is removed.

File: test/kubemark/start-kubemark.sh (hunk 2 above)
Comment (chenqianfzh) [reply test/kubemark/start-kubemark.sh:371]: These code created two test tenants, arktos/zeta, when we were trying with 2 TPs. Yet the code was changed to create variable number of TPs, where two test tenants may not be enough. We need the tester to create the test tenants on the fly. Therefore this section of code is removed.

Comment (pdgetrf) [reply test/kubemark/start-kubemark.sh:371]: thanks! consider this comment resolved.

File: cluster/gce/config-default.sh (hunk 1 above)
Comment (pdgetrf) [reply cluster/gce/config-default.sh:545]: thanks consider this comment resolved.

**Review by sonyafenge — APPROVED** [review]
/LGTM

**Comment by chenqianfzh** [comment]
> love seeing validation in the PR description. please also run the scale-up 100-node test to make sure of no regression.



> love seeing validation in the PR description. please also run the scale-up 100-node test to make sure of no regression.

Confirmed that scale-up scenario with a 100-node density run.

**Comment by pdgetrf** [comment]
> > love seeing validation in the PR description. please also run the scale-up 100-node test to make sure of no regression.
> 
> > love seeing validation in the PR description. please also run the scale-up 100-node test to make sure of no regression.
> 
> Confirmed that scale-up scenario with a 100-node density run.

thanks. please consider this comment resolved

**Review by pdgetrf — APPROVED** [review]
/lgtm

**Comment by chenqianfzh** [comment]
> love seeing validation in the PR description. please also run the scale-up 100-node test to make sure of no regression.

Confirmed that scale-up scenario also worked. Please check the updated PR description with the details.

**Pull request merged**